Makefile

```diff
@@ -126,7 +126,11 @@ clean:
 	rm -rf *.out
 	rm -rf parser
 
-test: checklist checkdep gotest explaintest gogenerate
+test: test_part_1 test_part_2
```

Comment:
We could add an annotation here explaining why it should be splited into 2 parts.